OffchainLabs/arbitrum-token-bridge
# fix: patch token name and symbol for USDC and USDCe

Extension of the PR https://github.com/OffchainLabs/arbitrum-token-bridge/pull/937
Covers all cases wherever we show USDC (symbol and name)

### Steps to test
- Log in via any account that has USDC transactions
- Search for USDC in token-search
- Check for the USDC symbol, name in Transfer panel and Tx history, they will now be chain specific.
